core.yaml

```diff
@@ -16,6 +16,9 @@ k8s:
     - name: gatekeeper-system
       disableIstioInjection: true
     - name: gitea
+    - name: glasnostic
```

Comment:
The glanostic is also out of scope. Please move Glasnostic related changes to different PR